src/components/CheckboxInput/components/Checkbox.tsx

```diff
@@ -0,0 +1,170 @@
+import React, {
```

Comment:
I have started to do `React.FC`, `React.WhateverThingINeed` in newer components. MInor style thing but perhaps worth discussing for consistency's sake. I am fine to continue with destructuring if that feels better to everybody. My thoughts on the dot syntax, it makes it clearer when something is coming from React and it also makes it easy to write new code without going back up and importing something every time its needed (or removing said import during a refactor)